python/tests/test_aac/context/test_definition_parser.py

```diff
@@ -18,6 +18,7 @@ def test_load_definitions_pass(self):
         self.assertEqual(len(context_definitions), 1)
         self.assertEqual(loaded_definitions, context_definitions)
         self.assertEqual(len(context_definitions[0].instance.fields), len(loaded_definitions[0].instance.fields))
+        self.assertTrue(loaded_definitions[0].source.is_loaded_in_context)
 
     def test_load_definitions_fail(self):
```

Comment:
Since this is a fail test should be able to do an `assertFalse` test like you did an `aasertTrue` test for the load success.